### DISCUSSION TITLE
< と >　で囲われた範囲でインラインコードがインラインコードとして扱われない

### DISCUSSION THREAD

**Discussion by kkkkan** [increments/qiita-discussions#413] [要望・アイデア・不具合]
<と>で囲われた範囲で、``を使用しても、インラインコードとして表示されないようです。
片側だけだとちゃんと表示されます。(空白文字を入れてみてもダメなようです。)

編集中のスクリーンショット↓
|![スクリーンショット 2023-01-08 15 49 39](https://user-images.githubusercontent.com/22609306/211184129-6a5f98fe-bcea-416d-8127-9731d252dd2e.png)|
|--|

実際に上記のものを投稿した記事↓
|![スクリーンショット 2023-01-08 15 54 52](https://user-images.githubusercontent.com/22609306/211184280-18188a66-bf0b-482a-bf4e-5b56eb793cf6.png)|
|--|

https://qiita.com/kkkkan/items/c57dee0fd4965105ad36

バグかな？と思うので、お手数ですがご確認いただけると嬉しいです。🙇🏻‍♀️(もし仕様でしたらすみません🙇🏻‍♀️)

**Comment by tanaka-asami415** [comment]
@kkkkan 
ご報告ありがとうございます！🙏
ご不便をおかけして、申し訳ありません。こちら調査いたします！

**Reply by kkkkan** [reply]
@tanaka-asami415 
ありがとうございます！よろしくお願いいたします。🙇🏻‍♀️

**Reply by kyntk** [reply]
@kkkkan おまたせいたしました🙏
こちら調査して原因はわかったのですが
他の開発優先度なども踏まえて対応を検討させていただきます🙇

**Reply by kkkkan** [reply]
@kyntk 
調査ありがとうございました！

> 他の開発優先度なども踏まえて対応を検討させていただきます🙇
承知しました。ご多忙のところ恐縮ですが、ご都合のいいタイミングで修正いただければと思います。🙏🏻
よろしくお願いいたします。🙇🏻‍♀️
